recipes/msdf-atlas-gen/all/test_package/conanfile.py

```diff
@@ -1,4 +1,4 @@
-from conans import ConanFile, tools
+from conan import ConanFile, tools
```

Comment:
This warning is because in general we try to only import specific tools we use in recipes instead of the whole package.